packages/core-data/src/selectors.js

```diff
@@ -180,3 +193,62 @@ export function isPreviewEmbedFallback( state, url ) {
 export function hasUploadPermissions( state ) {
 	return state.hasUploadPermissions;
 }
+
+/**
+ * Returns an attribute value of the current autosave revision for a post, or
+ * undefined if there is no autosave for the post.
+ *
+ * @param {Object} state         State tree.
+ * @param {Object} post          The parent post of the autosave.
+ * @param {string} attributeName Autosave attribute name.
+ *
+ * @return {*} Autosave attribute value.
+ */
+export function getAutosaveAttribute( state, post, attributeName ) {
+	if ( ! hasAutosave( state, post ) ) {
+		return;
+	}
+
+	const autosave = getAutosave( state, post );
+	if ( autosave.hasOwnProperty( attributeName ) ) {
+		return autosave[ attributeName ];
+	}
+}
+
+/**
+ * Returns the autosave that is a child of the provided post, if one exists.
+ *
+ * @param {Object} state State tree.
+ * @param {Object} post  The parent post of the autosave.
+ *
+ * @return {?Object} The autosave object, or undefined if there is none.
+ */
+export function getAutosave( state, post ) {
+	if ( state.autosave ) {
```

Comment:
When is `state.autosave` falsey?

Reply:
Thanks for catching that, removed the `if`.